### PR TITLE
Review wanted: Week6 weather app - Bea

### PR DESCRIPTION
**Pull request by bealun**
Super fun week but also super challenging! 
**(Can't keep this short, sorry in advance...)**

Filtered the forecast to get temp at 12pm (middle of the day) but that also means that the forecast doesn't update to new days 'til then. Worked to filter for middle of the night but the temp difference put me off so I made a judgement call and settled for this. Gotta know when to stop.

Added a small anitmation to my weather icon. Where the icon changes depending on degrees. If it's below 0 it will be a snowy cloud, if it's between 0 to +5 it will show a clouded sun and over 5 degrees it's sunny. Super proud of getting my if-statement to work and my animation too! Also put three European cities at the bottom of the page to compare todays weather to. Some stretch goals are checked!

If I had more time I would have liked to have the forecast grouped per day with a bit more info maybe. To have tomorrow in it's own card with more data and so on. The overall feeling is still that I'm very satisfied with my work this week and that I've learned more this week than with the projects before even though it's a rollercoster!

Deployed site: https://tender-goodall-aa3edb.netlify.com

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Hi! Is this project done and ready for a review? If yes - please update the PR title to "review wanted:" before your title you have now (you can edit your PR here on github). This is for the projects that's done before the deadline. If you change it to "review wanted" we will try and give you a review as soon as possible.

**Comment by bealun** [comment]
Sorry! Now updated title to ask for a review :)

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="shortcut icon" href="img/fav.png" type="image/x-icon">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+  <title>WEATHER FORECAST</title>
+</head>
+
+<body>
```
Comment (ericaglimsholt) [inline code/index.html]: I'm missing a `<h1>` in the `<body>`! You should always aim for having the biggest title in your page. It's very importen both for accessibility reasons and SEO purpose. But I understand if you don't have place for it, but you can actually have a `<h1>` that you make hidden with `visability: hidden;` or `opacity: 0;` in css! Then there is a `<h1>` but the user can't see it!

File: code/style.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
+html {
+  min-height: 100%;
+  border: 10px solid black;
+}
+
+body {
+  /* background: linear-gradient(90deg, rgba(131,193,207,1) 0%, rgb(176, 233, 224) 62%, rgb(208, 214, 218) 100%); */
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+p {
+  text-align: right;
+  margin: 3px 10px;
+}
+
+p:hover {
+  font-style: italic;
+}
```
Comment (ericaglimsholt) [inline code/style.css]: This is a nice touch!

File: code/weather.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
+// Week array
+const week = ['Sun', 'Mon', 'Tues', 'Wed', 'Thur', 'Fri', 'Sat']
+
+// DOM
```
Comment (ericaglimsholt) [inline code/weather.js]: I like that you have structure this like this!

File: code/weather.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    sunset.innerHTML = weather.sunset
+
+    if(weather.temp > 5) {
+        document.getElementById("whatWeather").src = warm
+    } else if(weather.temp > 0) {
+        document.getElementById("whatWeather").src = both
+    } else {
+        document.getElementById("whatWeather").src = cold
+    }
+    console.log(json)
```
Comment (ericaglimsholt) [inline code/weather.js]:
```suggestion
```

File: code/weather.js [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+  })
+
+// 5 day forecast local
+fetch(weatherForecast) 
+  .then(response => {
+    return response.json()
+
+  .then(json => {
+    console.log(json)
```
Comment (ericaglimsholt) [inline code/weather.js]:
```suggestion
```

File: code/weather.js [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    const filteredForecast = json.list.filter(item => item.dt_txt.includes('12:00'))
+    console.log(filteredForecast)
```
Comment (ericaglimsholt) [inline code/weather.js]:
```suggestion
```

File: code/weather.js [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+    weather.feels_like = (Math.round(json.main.feels_like * 10) / 10)
+    let nextCity = json.name
+
+    osloWeatherToday.innerHTML = `<p>${nextCity}: ${weather.temp}°C</p>`
+    })
+  })
+
+
+  
+// Animation for weather icon
```
Comment (ericaglimsholt) [inline code/weather.js]: I love that you have implemented an animation!

File: code/weather.js [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+function myIcon() {
+  var elem = document.getElementById("whatWeather")
```
Comment (ericaglimsholt) [inline code/weather.js]: This variable would suit nice next to the others DOM element!

File: code/weather.js [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+  var pos = 0
```
Comment (ericaglimsholt) [inline code/weather.js]: Be careful with  abbreviations. Some of them is hard to understand what it means. I think this one is kind of hard to tell what it is without reading how it is being used in the function. Also - the variables here could be set with `const` or `let`

**Pull request closed without merging**